src/commands/a32nx/fixinfo.ts

```diff
@@ -0,0 +1,26 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const FIX_INFO_URL = 'https://docs.flybywiresim.com/fbw-a32nx/assets/fixinfo/nd-orl-fixinfo2.png';
```

Comment:
We may need to change this URL. Not the image, maybe we can upload it to bot repo, or just copy and paste it as a discord link. Just in case the docs team change the path.